site/content/en/docs/Guides/Client SDKs/csharp.md

```diff
@@ -14,35 +14,35 @@ Check the [Client SDK Documentation]({{< relref "_index.md" >}}) for more detail
 
 | Area            | Action              | Implemented |
 |-----------------|---------------------|-------------|
-| Lifecycle       | Ready               | ✔️          |
-| Lifecycle       | Health              | ✔️          |
-| Lifecycle       | Reserve             | ✔️          |
-| Lifecycle       | Allocate            | ✔️          |
-| Lifecycle       | Shutdown            | ✔️          |
-| Configuration   | GetGameServer       | ✔️          |
-| Configuration   | WatchGameServer     | ✔️          |
-| Metadata        | SetAnnotation       | ✔️          |
-| Metadata        | SetLabel            | ✔️          |
-| Counters        | GetCounterCount     | ❌         |
-| Counters        | SetCounterCount     | ❌         |
-| Counters        | IncrementCounter    | ❌         |
-| Counters        | DecrementCounter    | ❌         |
-| Counters        | SetCounterCapacity  | ❌         |
-| Counters        | GetCounterCapacity  | ❌         |
-| Lists           | AppendListValue     | ❌         |
-| Lists           | DeleteListValue     | ❌         |
-| Lists           | SetListCapacity     | ❌         |
-| Lists           | GetListCapacity     | ❌         |
-| Lists           | ListContains        | ❌         |
-| Lists           | GetListLength       | ❌         |
-| Lists           | GetListValues       | ❌         |
-| Player Tracking | GetConnectedPlayers | ✔️          |
-| Player Tracking | GetPlayerCapacity   | ✔️          |
-| Player Tracking | GetPlayerCount      | ✔️          |
-| Player Tracking | IsPlayerConnected   | ✔️          |
-| Player Tracking | PlayerConnect       | ✔️          |
-| Player Tracking | PlayerDisconnect    | ✔️          |
-| Player Tracking | SetPlayerCapacity   | ✔️          |
+| Lifecycle       | Ready               | ✔️           |

```

Comment:
`feature` shortcodes please! (may be easiest to duplicate the table).

https://agones.dev/site/docs/contribute/#within-a-page

Reply:
Added that in